bee-network/src/network/host.rs

```diff
@@ -157,16 +172,16 @@ async fn process_swarm_event(
     }
 }
 
-async fn process_command(command: Command, swarm: &mut Swarm<SwarmBehavior>, peerlist: &PeerList) {
-    match command {
-        Command::DialPeer { peer_id } => {
-            if let Err(e) = dial_peer(swarm, peer_id, peerlist).await {
-                warn!("Failed to dial peer '...{}'. Cause: {}", alias!(peer_id), e);
-            }
-        }
+async fn process_icommand(icommand: Command, swarm: &mut Swarm<SwarmBehavior>, peerlist: &PeerList) {
```

Comment:
What does icommand stand for ?

Reply:
I've used `internal_command`, `internal_event` for a very long time, and I think it's too much noise.

Reply:
I really don't think it's noise when it's meaningful, icommand loses a lot of meaning IMO

Reply:
But yeah, we kinda know we disagree on these things, just wanted to voice my opinion

Reply:
I really am very much undecided on this, but if there's someone who would prefer the written out version, then I'ld stick to what I had before. Should I change back?

Reply:
As you want. I personally have no issue with the long version `internal_command` and have no issue with simply `command`. I just think that `icommand` raises more questions than it should.

Reply:
alright, will not do it then.